### PR TITLE
Rrogahn/deployment tutorial

### PR DESCRIPTION
**Pull request by RickRogahn**
Updating the deployment "getting started" page.

### CODE REVIEW

File: academic-services/knowledge-exploration-service/get-started-create-api-instances.md [hunk 1]
```diff
@@ -5,24 +5,79 @@ ms.topic: tutorial
...
+
+- Replace "[your_storage_account_name]" with the name of the storage account you downloaded the scripts from above.
+
+At this point the tool will take care of creating all of the required resources and deploying your instance of MAKES.  This process can take a long time.  During the initial run it can take up to 3 hours to complete.  During the first run the script generates some resources that you only need to create once.  Subsequent deployments will not take as long if you specify existing resources.  See the [Command line Reference](reference-makes-command-line-tool.md) for more details.  
+
+Depending on the region your storage account is in and the region you are deploying to, deployment may take longer as the default indexes are quite large and need to be copied.  By default, the tool deploys to the WestUS region of Azure.  You can change this by adding parameters to the script.  For a reference of all the available parameters type:
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/get-started-create-api-instances.md]: I think this section should be tagged with as an "important" section. I'd suggest the following:

[!NOTE] To achieve the fastest instance start times ensure that all resources (storage account, virtual machine scale set, etc.) are located in the same region. The "--Region" parameter controls which region new resources are created in. Visit the [Command Line Reference](reference-makes-command-line-tool.md) section for full details on this and other parameters.

File: academic-services/knowledge-exploration-service/get-started-create-api-instances.md [hunk 2]
```diff
@@ -5,24 +5,79 @@ ms.topic: tutorial
...
+```cmd
+kesm.exe --help
+
+// For command specific help; type the command, appending '--help'
+
+kesm.exe DeployHost --help
 ```
 
-## Deploy MAKES API instances
+Or, you can visit the [kesm.exe Command Line Reference](reference-makes-command-line-tool.md).
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/get-started-create-api-instances.md]: When I was using the tool I made a stupid mistake by using a hyphen in my host name. This caused me to waste 30 minutes as the VM template was created but then deployment immediately failed as host names can't include hyphens.

We should probably add an "[!IMPORTANT]" section above that warns users of this pitfall (or modify the tool to make this check up-front)

File: academic-services/knowledge-exploration-service/get-started-create-api-instances.md [hunk 3]
```diff
@@ -5,24 +5,79 @@ ms.topic: tutorial
...
 
 ## Prerequisites
 
-- MAKES release
-- MAKES indexes/grammar
+- Microsoft Academic Knowledge Exploration Service (MAKES) subscription
 - Azure subscription
 
-## Deploy MAKES API hosting resources
+## Verify the current release
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/get-started-create-api-instances.md]: For step-by-steps we should use numbered lists as it helps users keep track of where they are.

File: academic-services/knowledge-exploration-service/get-started-create-api-instances.md (hunk 2 above)
Comment (RickRogahn) [reply academic-services/knowledge-exploration-service/get-started-create-api-instances.md]: I actually had Alvin change the validation on the parameters to happen at initial execution.  There is a PR out for this change.  I ran into this as well :|.

File: academic-services/knowledge-exploration-service/index.yml [hunk 4]
```diff
@@ -24,7 +24,7 @@ sections:
     style: unordered
     items:
     - html: <a href="get-started-setup-provisioning">Sign up for Microsoft Academic Knowledge Exploration Service(MAKES)</a>
-    - html: <a href="get-started-create-api-instances">Deploy an MAKES API instance using MAKES command line tool</a>
+    - html: <a href="get-started-create-api-instances">Deploy a basic MAKES instance using the provided scripts</a>
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/index.yml]: >basic  [](start = 64, length = 6)

I'd remove "basic" as it implies we have an advanced

File: academic-services/knowledge-exploration-service/get-started-create-api-instances.md (hunk 3 above)
Comment (DarrinEide) [reply academic-services/knowledge-exploration-service/get-started-create-api-instances.md]: Sorry, I actually intended this for the bulleted lists inside the sections

---
In reply to: [370777199](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/90#discussion_r370777199) [](ancestors = 370777199)

File: academic-services/knowledge-exploration-service/get-started-create-api-instances.md [hunk 5]
```diff
@@ -5,24 +5,79 @@ ms.topic: tutorial
...
+
+![Download kesm.zip](media/get-started-download-kesm.png)
+
+Save this file to a local folder.
+
+## Unzip the script package
+
+Once the file has been downloaded you will need to extract the contents.  Right click on the kesm.zip file.  If you are using the Windows operating system select "Extract All" from the context menu.  If you are using linux, select "Extract Here" or "Unzip Here".  If you are using a Mac, double-click the zip file and it will decompress into the same folder.
+
+If this has completed successfully you will have extracted a single file named kesm.exe.
```
[the remark this answers is not in the document]
Comment (DarrinEide) [reply academic-services/knowledge-exploration-service/get-started-create-api-instances.md]: I think it's important to be accurate one what gets extracted

---
In reply to: [370752700](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/90#discussion_r370752700) [](ancestors = 370752700)

File: academic-services/knowledge-exploration-service/get-started-create-api-instances.md (hunk 1 above)
Comment (DarrinEide) [reply academic-services/knowledge-exploration-service/get-started-create-api-instances.md]: I was wrong on the formatting for these, sorry!

It needs to be:
> [!NOTE|IMPORTANT|WARNING|INFO]
> Message

---
In reply to: [370773816](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/90#discussion_r370773816) [](ancestors = 370773816)

File: academic-services/knowledge-exploration-service/get-started-create-api-instances.md (hunk 2 above)
Comment (DarrinEide) [reply academic-services/knowledge-exploration-service/get-started-create-api-instances.md]: Excellent!@

---
In reply to: [370872227](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/90#discussion_r370872227) [](ancestors = 370872227)

File: academic-services/knowledge-exploration-service/get-started-create-api-instances.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (RickRogahn) [reply academic-services/knowledge-exploration-service/get-started-create-api-instances.md]: Updated

---
In reply to: [371957761](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/90#discussion_r371957761) [](ancestors = 371957761,370752700)

**Pull request merged**